src/flexmeasures_client/response_handling.py

```diff
@@ -22,11 +22,19 @@ async def check_response(self: FlexMeasuresClient, response, polling_step: int):
     status = response.status
     payload = await response.json()
     headers = response.headers
-    print(payload)
-    print(status)
+    logging.debug(status)
+    logging.debug(payload)
+    logging.debug(headers)
     if status < 300:
         pass
     elif status == 401:
+        message = f"""Authentication failed with"
+        status: {status}
+        headers: {headers}
+        payload: {payload}.
+        Re-authenticating!
+        """
+        logging.info(message)
```

Comment:
I think this one should be a warning instead.

Reply:
this is normal behavior. When the access_token expires a new access_token should be fetched. Could be in debug but think this would be useful as an info message..